packages/authentication/package.json

```diff
@@ -4,9 +4,10 @@
   "description": "A LoopBack component for authentication support.",
   "scripts": {
     "acceptance": "mocha --opts ../../test/mocha.opts 'test/acceptance/**/*.ts'",
-    "build": "npm run build:lib && npm run build:lib6",
+    "build": "npm run build:lib && npm run build:lib6 && npm run build:apidocs",
```

Comment:
@bajtos we tried all that you said, that is creating documentation on-the-fly for TS as well, but for that we didnt have sources, and as you suggested we added sources to the build dir, but that didn't help because it needed entire build env. You cannot "lerna bootstrap" the env in npm download like you do for github download. In the npm world all packages are independent, in github we tie them together using lerna.